packages/react-components/src/components/VideoTile.tsx

```diff
@@ -219,24 +228,25 @@ export const VideoTile = (props: VideoTileProps): JSX.Element => {
           })}
         />
 
-        {isVideoRendered ? (
+        <Stack className={mergeStyles(videoContainerStyles, { zIndex: _VIDEO_TILE_Z_INDEX })}>
+          {onRenderPlaceholder ? (
+            onRenderPlaceholder(userId ?? '', placeholderOptions, DefaultPlaceholder)
+          ) : (
+            <DefaultPlaceholder {...placeholderOptions} />
+          )}
+        </Stack>
+
+        {isVideoRendered && (
```

Comment:
Will this allow for us to provide a custom tile overlay for when the user is being called in? the logic of needing the video rendered is confusing me a little

Reply:
yup! so its actually based on whether there is a video there or our new state tile. is that right?

Reply:
So it shouldn't matter,
as long as the renderElement prop has been set with a JSX element, this piece should render

Reply:
Correct, as long as the prop has a JSX element.